fitbenchmarking/fitbenchmarking
# Adding Jacobian class

#### Description of Work

Fixes #479, fixes #487

Adds new Jacobian classes which implement SciPy's numerical Jacobians. The docs can be found [here](https://fitbenchmarking.readthedocs.io/en/479-jacobian_class/)

#### Testing Instructions

1. Check docs are clear
2. Test the Jacobian options

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
